From 60f46de79d9bb8d45aafde22dcf8ab179c9bd843 Mon Sep 17 00:00:00 2001
From: aspeedyh <yh_chung@aspeedtech.com>
Date: Thu, 13 Nov 2025 14:00:40 +0800
Subject: [PATCH] configs: use SPL_ prefix for SPL compatibility

add a SPL_ prefix to CONFIG_ASPEED_EDAF_BRIDGE to ensure edaf_bridge
driver is built in both U-Boot and SPL.

Signed-off-by: aspeedyh <yh_chung@aspeedtech.com>
Change-Id: I2c4dd6ddedea0e3f82811406afec588f627658db
(cherry picked from commit 3443bbc6d497c5c58e24848749e6cce1c21c9997)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/dts/ast2700-evb.dts | 5 -----
 drivers/misc/Makefile        | 2 +-
 2 files changed, 1 insertion(+), 6 deletions(-)

diff --git a/arch/arm/dts/ast2700-evb.dts b/arch/arm/dts/ast2700-evb.dts
index 4edbc2a8e059..3890e0a03a89 100644
--- a/arch/arm/dts/ast2700-evb.dts
+++ b/arch/arm/dts/ast2700-evb.dts
@@ -199,12 +199,7 @@
 &edaf_bridge0 {
 	status = "okay";
 	espi-base = <0x0 0x14c05000>;
-#if 0 //DDR mode using edaf0 reserved memory
-	edaf-ddr-mode;
-	mem-base = <&edaf0>;
-#else
 	mem-base = <0x1 0x80000000>; //SPI0
-#endif
 	ctl-base = <0x0 0x14010000>;
 	cmd-read = <0x130401>;
 	cmd-write-enable = <0x6041e>;
diff --git a/drivers/misc/Makefile b/drivers/misc/Makefile
index a187155ce63f..e8cd9d72481b 100644
--- a/drivers/misc/Makefile
+++ b/drivers/misc/Makefile
@@ -92,7 +92,7 @@ obj-$(CONFIG_SL28CPLD) += sl28cpld.o
 obj-$(CONFIG_$(SPL_)ASPEED_DP) += aspeed_dp.o
 obj-$(CONFIG_$(SPL_)ASPEED_VGA) += aspeed_vga.o
 obj-$(CONFIG_$(SPL_)ASPEED_VBIOS) += aspeed_vbios.o
-obj-$(CONFIG_ASPEED_EDAF_BRIDGE) += aspeed_edaf_bridge.o
+obj-$(CONFIG_$(SPL_)ASPEED_EDAF_BRIDGE) += aspeed_edaf_bridge.o
 obj-$(CONFIG_SPL_SLI_AST2700) += sli_ast2700.o
 obj-$(CONFIG_SPL_PRIVILEGE_CTRL) += aspeed_prictrl.o aspeed_sram_prictrl.o
 obj-$(CONFIG_SPL_LTPI_AST2700) += ltpi_ast2700.o
-- 
2.34.1

